openlibrary/templates/lists/widget.html

```diff
@@ -168,7 +168,7 @@
                   </form>
               </div>
 
-              $if not readinglog_only:
+              $if not readinglogonly:
```

Comment:
typo compared to var name below

Reply:
intentionally -- global template var readinglog_only was defined in `$def with (readinglog_only=...)`. The name within the `$jsdef` needs to be different than this var